src/main.cpp

```diff
@@ -51,20 +53,27 @@ float calculate_frequency(float velocity)
 	return (u + velocity) * 2 * M_PI / L;
 }
 
-void set_inverter_pins_(bool v, unsigned pin_H, unsigned pin_L)
+void set_logic_pin_(bool v) {
+	gpio_put(pinU_H, v);
+	gpio_put(pinU_L, 1);
```

Comment:
Thought: was considering that maintaining the enable pin on each call seems unnecessary, but perhaps it will help keep the driving frequency similar between the two modes?

Reply:
From my point of view, I thought doing this would make this more organized.